CITATIONS.md

```diff
@@ -60,6 +60,15 @@
 * [ITSx](https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.12073)
     > Bengtsson-Palme, J., Ryberg, M., Hartmann, M., Branco, S., Wang, Z., Godhe, A., De Wit, P., Sánchez-García, M., Ebersberger, I., de Sousa, F., Amend, A., Jumpponen, A., Unterseher, M., Kristiansson, E., Abarenkov, K., Bertrand, Y.J.K., Sanli, K., Eriksson, K.M., Vik, U., Veldre, V. and Nilsson, R.H.. Improved software detection and extraction of ITS1 and ITS2 from ribosomal ITS sequences of fungi and other eukaryotes for analysis of environmental sequencing data. Methods Ecol Evol 2013, 4: 914-919. doi: 10.1111/2041-210X.12073.
 
+* [PICRUSt2](https://pubmed.ncbi.nlm.nih.gov/32483366/)
+    > Douglas GM, Maffei VJ, Zaneveld JR, Yurgel SN, Brown JR, Taylor CM, Huttenhower C, Langille MGI. PICRUSt2 for prediction of metagenome functions. Nat Biotechnol. 2020 Jun;38(6):685-688. doi: 10.1038/s41587-020-0548-6. PMID: 32483366; PMCID: PMC7365738.
+
+* PICRUSt2 is using in this pipeline by default [EPA-ng](https://pubmed.ncbi.nlm.nih.gov/30165689/)
```

Comment:
Grammar is not correct in "is using in", do you mean "is used in" or maybe "is using"? If the latter, maybe "pipeline" is not correct, maybe just "PICRUSt2 is by default using [EPA-ng]"?

Reply:
Thanks, will change!